pkg/util/github/github.go

```diff
@@ -125,7 +125,7 @@ func DownloadOpenShiftReleaseBinary(binaryType OpenShiftBinaryType, osType minis
 		return errors.Wrap(err, fmt.Sprintf("Cannot download OpenShift release asset %d", assetID))
 	}
 	if len(url) > 0 {
-		glog.V(2).Infof("Downloading %s %s\n", binaryType.String(), *release.TagName)
+		fmt.Println(fmt.Sprintf("Downloading OpenShift client binary %s with version '%s'", binaryType.String(), *release.TagName))
```

Comment:
The message becomes wrong here. _DownloadOpenShiftReleaseBinary_ can be used to download _openshift_ or _oc_. By now we only use it for _oc_ apparently, but still the method does allow to download _openshift_, in which case the message becomes wrong. So, if we want to change this, it would be more along the lines:

"Downloading binary %s  in version %s\n", binaryType.String(), *release.TagName)"
or
"Downloading OpenShift binary  %s  in version %s\n", binaryType.String(), *release.TagName)"

Also, I think it is "you are downloading something _in_ a specific version" not _with_.